doc/releasing.md

```diff
@@ -32,11 +32,22 @@ Do the following when releasing:
         --secret silk_client_id=${silk_client_id} \
         --secret silk_client_secret=${silk_client_secret} \
         +sbom-download \
-        --out cyclonedx.augmented.sbom.json
+        --out cyclonedx.augmented.sbom.json \
```

Comment:
Do we know if there is guidance on how this file should be named, i.e. for consistency with other projects/repos?

Reply:
Not AFAIK. Though I see a variety of ways listed in [CVM Readme > MongoDB's Asset Groups](https://docs.google.com/document/d/18DZhrMvdxlFJJS7KaZB1gBNIKispX1W4rlTD-16yGI8/edit#bookmark=id.4wujkv7vmbgy) (`third_party/sbom/cyclonedx.sbom.json`, `docs/releases/v2.2.2/linux-amd64.sbom.json`, `sbom.json`).